### PR TITLE
Stalin-Sort in MIPS Assembly

### PR DESCRIPTION
**Pull request by cmiram**
Tested using the MARS simulator.

http://courses.missouristate.edu/kenvollmar/mars/

### CODE REVIEW

File: mips/stalin-sort.s [hunk 1]
```diff
@@ -0,0 +1,118 @@
+	.data
+	
+unsortedArray:
+	.word 1, 2, 4, 3, 6, 8, 0, 9, 5, 7
```
Comment (gustavo-depaula) [inline mips/stalin-sort.s]: what happens with repeated numbers like `.word 1, 2, 4, 3, 4, 8, 0, 9, 5, 7`?

Comment (cmiram) [reply mips/stalin-sort.s]: It keeps the repeats as shown below. Check line 64 where branches to skip adding the unsorted element only if it is less than the most recently added element to the sorted list.

image

**Comment by gabrielcarneiro97** [comment]
Thanks for your contribution!

File: mips/stalin-sort.s (hunk 1 above)
Comment (gustavo-depaula) [reply mips/stalin-sort.s]: it's been ages since I saw a MIPS assembly code. great work!

**Pull request merged**